components/docs.html

```diff
@@ -288,6 +288,176 @@ <h1 class="comp-lib-avatar-color">Alert!</h1>
                     
                     <!-- alert section end -->
 
+                     <!-- button section start -->
+
+
+                     <div class="Button common-container" id="comp-button">
+
+                        <div class="common-wrapper">
+                            <h1 class="comp-lib-avatar-color">Button</h1>
+                            <div class="comp-lib-avatar-content_a">Use <span class="comp-lib-span">HiFi's</span> custom button styles for actions in forms, dialogs, and more with support for multiple sizes, states, and more.</div>
+                            <h3 class="button-sub-heading">Example of Icon Button with Text</h3>
+                            
+                            <div class="main-icon">
+                                <button class="icon-btn">
+                                    <span class="material-icons icon-color">home</span>
+                                    <span class="icon-text">Home</span>
+                                </button>
+                                <button class="icon-btn">
+                                    <span class="material-icons icon-color">menu</span>
```

Comment:
Just a Suggestion. While naming the icon button you could write "btn-icon" instead of "icon-btn" to make it look more consistent with the code.

Reply:
Also, some of the buttons do not have a cursor as a pointer on hover, fix it.